src/pyVector3.py

```diff
@@ -0,0 +1,51 @@
+class pyvector3:
```

Comment:
Sorry, but again, the same question: what is the purpose of this class?
What was the problem to use `np.array` directly?

Reply:
done by removing pyvector3